evalml/demos/fraud.py

```diff
@@ -23,8 +25,9 @@ def load_fraud(n_rows=None, verbose=True, return_pandas=False):
                      target="fraud",
                      n_rows=n_rows,
                      verbose=verbose)
-    X = X.set_types({"provider": "Categorical", "region": "Categorical"})
-
     if return_pandas:
-        return X.to_dataframe(), y.to_series()
+        return X, y
```

Comment:
JW, do you think we still need the `return_pandas` parameter now? Is it okay to always return a WW init version since users can still just use the pandas data structures if they wanted to? :o